deploy/charts/codegate/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "codegate.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "codegate.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
```

Comment:
have we tried this with multiple replicas yet? This might conflict with the single-file sqlite database we have. Maybe we should limit it to 1.

Reply:
Nope not with multiple, defo should limit to `1`. Deployment with a PVC also forces the constraint of running it with `1`. Unless we use stateful sets but then the single file db isn't really a thing then